elrond-wasm-node/src/ext.rs

```diff
@@ -76,7 +76,10 @@ extern {
     
     // crypto API
     fn sha256(dataOffset: *const u8, length: i32, resultOffset: *mut u8) -> i32;
-    fn keccak256(dataOffset: *const u8, length: i32, resultOffset: *mut u8) -> i32;
+	fn keccak256(dataOffset: *const u8, length: i32, resultOffset: *mut u8) -> i32;
```

Comment:
cd into elrond-wasm-node and also call `cargo fmt` there to get everything nicely aligned